filters/ThirdParty/filter_227_List-KR/template.txt

```diff
@@ -1,4 +1,4 @@
 !
 @include "https://raw.githubusercontent.com/List-KR/List-KR/master/filter-AdGuard.txt" /exclude="../../exclusions.txt"
-! Javascript rules
-tv.nate.com#%#AG_onLoad(() => { var BuildObs, PlayerObs; let PlayerCall = (mutationList, observer) => { mutationList.forEach((e) => { if (Array.from(e.target.classList).some((e) => { return e === "ntv-ad-playing" })) { player.trigger("adended"); PlayerObs.disconnect() }})}; let BuildCall = (mutationList, observer) => { mutationList.forEach((e) => { Array.from(e.addedNodes).filter((o) => { return o.id === "natePlayer" }).forEach((k) => { PlayerObs = new MutationObserver(PlayerCall); PlayerObs.observe(k, { attributes: true }); BuildObs.disconnect() })})}; BuildObs = new MutationObserver(BuildCall); BuildObs.observe(document.querySelector(".player-wrapper"), { childList: true }) })
\ No newline at end of file
+! This file contains the rules that are needed to handle problem sites
```

Comment:
It was meant to add this description to the `trusted-rules.txt`, not to the `template.txt`.

> are needed to handle problem sites

not only problematic. All, where applied trusted rules.